shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/App.kt

```diff
@@ -1,13 +1,16 @@
 package org.jetbrains.kotlinconf
 
+import URLs
```

Comment:
would be nice to move it under our package

Reply:
Good catch!